packages/design-system/src/styles/components/_Button.scss

```diff
@@ -27,7 +27,7 @@
   &.ds-c-button--focus {
     @if $ds-include-focus-styles {
       background-color: $focus-color;
-      border-color: $color-base;
+      border-color: $color-primary-darker;
```

Comment:
Isnt this updating the styling for when the global focus style flag is on? I dont think we need to be updating that for this PR right?

Reply:
Yep that's right - changing it back. Thanks for the catch